### DISCUSSION TITLE
Version 2.0.2

### DISCUSSION THREAD

**Discussion by EntraptaJ** [K-FOSS/TS-ESNode#227] [General]
@yoursunny I discovered a minor issue with some modules, in fixing the bug, I realized my fix could very easily break many existing applications or external modules. Prior to 2.0.0 modules such as Winston would error upon import. Please be careful when updating to `2.0.0`. It will allow usage of several modules, but has a chance of breaking existing usages.

**Comment by yoursunny** [comment]
I tried 2.0.2 and no ill-effects are found.

**Reply by EntraptaJ** [reply]
Awesome.
